lib/food_truck.rb

```diff
@@ -0,0 +1,21 @@
+class FoodTruck
+  attr_reader :name, :inventory
+
+  def initialize(name)
+    @name = name
+    @inventory = {}
+  end
+
+  def check_stock(item)
+    @inventory[item] || 0
+  end
+
+  def stock(item, quantity)
+    @inventory[item] ||= 0
+    @inventory[item] += quantity
```

Comment:
Yeah, this works, however you could also have set the @inventory hash on line 6 to have a default value of 0 to reduce the need for memoization here. Either way though